### DISCUSSION TITLE
error in publishing npm package

### DISCUSSION THREAD

**Discussion by miminerd** [npm/feedback#238] [CLI] [answered]
Hello, 
I got this error when I try to publish my npm package, though I published before and never happened.

npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.

this was resolved by specifying 'version ' in package.json or confirming the email registry, ...
can you please detail  the  error log error when there are some.

Thank you and have a nice day

**Comment by MylesBorins** [comment, marked as answer]
Hey @miminerd thanks for the heads up! Would you be able to open an issue at https://github.com/npm/cli outlining the issue you ran into?
